configs/urls.ts

```diff
@@ -1,2 +1,3 @@
 export const twitchTvApiEndpoint = 'https://api.twitch.tv/helix';
 export const logsTfUploadEndpoint = 'https://logs.tf/upload';
+export const etf2lApiEndpoint = 'http://api.etf2l.org';
```

Comment:
Isn't this the wrong API endpoint, didn't supra want you to move to API v2?

 https://api-v2.etf2l.org/

Reply:
It's a refactor, I'll update the endpoint to v2 in another pull request.